boot/src/boot_hedge/core.clj

```diff
@@ -28,3 +30,20 @@
     (case cloud
       :aws (require '[boot-hedge.aws.core :as aws])
       :azure (require '[boot-hedge.azure.core :as azure]))))
+
+(defn find-version [version line]
+  (if (clojure.string/starts-with? line "version=")
+    (last (clojure.string/split line #"="))
+    version))
+
+(defn get-hedge-boot-version
+  "Return the version of boot-hedge."
+  []
+  (let [props (clojure.java.io/resource "META-INF/maven/siili/boot-hedge/pom.properties")
```

Comment:
Clojure can read Properties files in with
```clj
(require '[clojure.java.io :refer (reader)])
(def props (java.util.Properties.))
(.load props (reader "fruitcolors.properties"))
```
This uses Java's native Properties class which supports comments and all kinds of "fancy" stuff like that which ensures you'll get the correct value. See also https://github.com/clojure-cookbook/clojure-cookbook/blob/master/04_local-io/4-18_read-property-file.asciidoc